.bumpversion.cfg

```diff
@@ -1,12 +1,92 @@
 [bumpversion]
-current_version = 0.3.1
+current_version = 0.3.2
 commit = True
 tag = True
 
-[bumpversion:file:setup.cfg]
-search = version = {current_version}
-replace = version = {new_version}
+[bumpversion:file:pyproject.toml]
+search = version = "{current_version}"
+replace = version = "{new_version}"
 
 [bumpversion:file:src/cryoemservices/__init__.py]
 search = __version__ = "{current_version}"
 replace = __version__ = "{new_version}"
+
+[bumpversion:file:/home/wjr38212/Workspace/repos/cryoem-services/Helm/Chart.yaml]
```

Comment:
Automatic replacement sometimes gives unexpected consequences? (Issue repeats for the subcharts too)